CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -9,9 +9,20 @@
 
 See [code changes](https://github.com/aws/aws-k8s-tester/compare/v1.4.5...v1.4.6).
 
+### `eksconfig`
+
+- Add [`SkipDeleteClusterAndNodes` to skip EKS cluster and nodes deletion](https://github.com/aws/aws-k8s-tester/commit/565635179860b4832b64cbe3b39fdbe1c12b1ae1).
```

Comment:
Consider calling this "Attach to cluster"

Reply:
Instead of skipping, can we idempotently check that it already exists and just succeed?

Reply:
Thinking even higher level, how much effort would it be to make everything within AKT idempotent, so that we could modify the yaml config and it would simply execute the deltas?

Reply:
Do you mean skipping creation? Creation is already done here

https://github.com/aws/aws-k8s-tester/blob/4cad9b32c68f212265af8d825e586847104cfb50/eks/cluster/cluster.go#L288-L303

which means you can run `create cluster` command multiple times and it will automatically skip if already created.